src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/Resources/Strings.resx

```diff
@@ -456,4 +462,34 @@
   <data name="RequiresAllowUnsafeBlocksTitle" xml:space="preserve">
     <value>LibraryImportAttribute requires unsafe code.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="StatefulMarshallerRequiresFromManagedDescription" xml:space="preserve">
+    <value>A stateful marshaller that supports a managed-to-unmanaged marshal mode must provide a 'FromManaged' instance method that takes the managed value as a parameter and returns 'void'.</value>
```

Comment:
We have varied phrasing referring to different shapes/support in the descriptions - let's choose one. Examples I see:
1. supports a managed-to-unmanaged marshal mode
2. supports marshalling from managed to unmanaged
3. 'Managed to Unmanaged with Caller-Allocated Buffer' shape is used

I prefer (2) over (1), since 'managed-to-unmanaged marshal mode' could be interpreted as `MarshalMode.ManagedToUnmanaged*` instead. I think giving the message the specific `MarshalMode` is sufficient for tying it to what exactly the user specified to indicate 'supports marshalling from ... to ...'. (3) seems a bit out of place (maybe unless we give the shapes official names and link to doc with them), so I think it might be clearer to spell out as 'supports marshalling from ... to ... with a caller-allocated buffer'